cinema/tests/test_movie_api.py

```diff
@@ -10,6 +10,7 @@
 from rest_framework import status
 
 from cinema.models import Movie, MovieSession, CinemaHall, Genre, Actor
+from cinema.serializers import MovieListSerializer, MovieDetailSerializer
```

Comment:
The import statement for `MovieListSerializer` and `MovieDetailSerializer` is correct and ensures that the serializers are available for use in the test cases.